server/src/main/java/com/google/musicanalysis/types/Tuple.java

```diff
@@ -0,0 +1,13 @@
+package com.google.musicanalysis.types;
+
+import java.util.HashMap;
+
+public class Tuple {
```

Comment:
This isn't really a generic Tuple. It's a string-to-string map with a magnitude and score. Can we create a better class that encapsulates this info all in one and does so more descriptively?

Reply:
I'm not sure I completely follow. This resolve commit just renames Tuple to Analysis pair, but what do you mean by more descriptively?

Reply:
The renaming is mostly what I wanted. Potentially splitting the MagnitudeAndScore object to just be parameters in this class.